model/task/db.go

```diff
@@ -25,6 +25,8 @@ const (
 	OldCollection = "old_tasks"
 )
 
+var annotationLookupDeprecationTime = time.Date(2024, time.January, 18, 23, 0, 0, 0, time.UTC)
```

Comment:
I'm assuming this will be updated to the date this is merged? Just wanted to flag.

Reply:
Thanks, updated to today.